Automattic/wc-calypso-bridge
# Add Woocomerce Onboarding task to set up Stripe

### Changes proposed in this Pull Request:



Closes https://github.com/Automattic/dotcom-forge/issues/8257

This PR adds the onboarding task to get paid with Stripe and removes Woo payments.

<img width="837" alt="Screenshot 2024-07-18 at 10 59 07 PM" src="https://github.com/user-attachments/assets/8bf7c74c-6074-4462-ad0b-d36cf4d4a9d2">





### How to test the changes in this Pull Request:



1. Create a free trial eCommerce site for Development following the steps described in this https://hostedwoo.wordpress.com/2023/02/09/setting-up-a-free-ecommerce-trial-site-for-development/
2. Open atomic `_cli` and install Stripe plugin `wp wpcomsh plugin use-managed woocommerce-gateway-stripe --activate`
3. Also, from the `_cli` Update the `woocommerce_onboarding_profile` option by using the following command `wp option patch insert woocommerce_onboarding_profile partner stripe`
4. Check out this branch on your local environment and synchronize the files with the plugin on atomic by following these steps. https://teameaglep2.wordpress.com/how-to-guides/working-with-the-woocommerce-calypso-bridge/
5. Go to the `wc-admin` page `/wp-admin/admin.php?page=wc-admin`
6. Confirm that the `Get paid with Stripe` task is displayed.
7. You should be redirected to Stripe Connect.




### Other information:




### FOR PR REVIEWER ONLY:
